### PR TITLE
Tr 6 check product comparisons

### PR DESCRIPTION
**Pull request by Obibus**

### CODE REVIEW

File: src/test/java/pages/CartPage.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+import elements.CartPageElements;
+import org.openqa.selenium.WebDriver;
+
+public class CartPage extends CartPageElements {
+    public CartPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public CartPage clickDeleteButton() {
```
Comment (kimkaminska) [inline src/test/java/pages/CartPage.java:11]: Please, add @Step

File: src/test/java/pages/CartPage.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        getDeleteButton().click();
+        return this;
+    }
+
+    public CartPage clickCloseButton() {
```
Comment (kimkaminska) [inline src/test/java/pages/CartPage.java:16]: Please, add steps everywhere

File: src/test/java/tests/CheckCartTest.java [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+                .clickConfirmationOfLocationButton()
+                .sendKeysSearchField(IPHONE)
+                .clickSearchButton();
+
+        shopPage
+                .moveToCardInfo()
+                .clickBuyButton();
+
+        String selectProductPrice = shopPage.getTextProductPrice();
+        softAssert.assertEquals(selectProductPrice, shopPage.getTextProductPriceInCart());
```
Comment (kimkaminska) [inline src/test/java/tests/CheckCartTest.java:33]: Please, add empty space before softAssert

File: src/test/java/tests/CheckCartTest.java [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+import base.AbstractBaseTest;
+import jdk.jfr.Description;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import pages.CartPage;
+import pages.HomePage;
+import pages.ShopPage;
+
+public class CheckCartTest extends AbstractBaseTest {
+    public static final String MY_ACCOUNT_PAGE_LINK = "https://epicentrk.ua/ua/personal/order/make/";
```
Comment (kimkaminska) [inline src/test/java/tests/CheckCartTest.java:12]: Please, add empty line before constants

File: src/test/java/tests/ProductComparisonTest.java [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+import base.AbstractBaseTest;
+import jdk.jfr.Description;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.ComparePage;
+import pages.HomePage;
+import pages.ShopPage;
+
+public class ProductComparisonTest extends AbstractBaseTest {
+    public static final String IPHONE = "iPhone 14 Pro Max";
```
Comment (kimkaminska) [inline src/test/java/tests/ProductComparisonTest.java:12]: Please, add empty line before constants

File: src/test/java/tests/ProductComparisonTest.java [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+        shopPage
+                .clickSecondItemInListCompare()
+                .clickCompareIcon();
+
+        comparePage
+                .clickCompareButton()
+                .clickClearList();
+
+        String pageSource = getPageSource();
+        Assert.assertTrue(pageSource.contains("Вибачте, але Ви ще нічого не додали в порівняння"), "Вибачте, але Ви ще нічого не додали в порівняння");
```
Comment (kimkaminska) [inline src/test/java/tests/ProductComparisonTest.java:56]: Please, add empty line before Assert

**Review by VadymTT — APPROVED** [review]

**Review by s1nL1g — APPROVED** [review]

**Pull request merged**